install/operator/.lib.sh

```diff
@@ -135,7 +135,7 @@ build_operator()
           OPTS="$OPTS '$i'"
         done
 
-        docker build -t "${BUILDER_IMAGE_NAME}" . -f - <<EODockerfile
+        cat > Dockerfile.tmp <<EODockerfile
```

Comment:
This is a workaround for #6315 , will remove once the fix for that is merged

Reply:
I think https://github.com/syndesisio/syndesis/pull/6322/files has a more complete solution for this issue, maybe we should remove the bits here

Reply:
Yep, will do